infra/resources/_modules/cosmos/cosmos_account.tf

```diff
@@ -0,0 +1,47 @@
+module "cosmos_account_wallet" {
```

Comment:
@pasqualedevita Could you please confirm we would like to use CMK?

@balanza In that case, we need to add a KeyVault reference using the `key_vault_key_id` property (already supported by this module) and reference the key. Please also note [these suggestions from HashiCopr](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/cosmosdb_account#key_vault_key_id)

Reply:
for me it's ok to use a key vault CMK, but be sure to set purge protection on keyvault at least 90 days

@pp-ps are u agree?

Reply:
Agreed, we will spread more information on CMKs in the following months.